FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/PrivateKeysViewModel.kt

```diff
@@ -406,43 +406,13 @@ class PrivateKeysViewModel(application: Application) : AccountViewModel(applicat
     }
   }
 
-  private suspend fun savePrivateKeyToDatabase(
-    accountEntity: AccountEntity,
-    pgpKeyDetails: PgpKeyDetails,
-    passphrase: String
-  ) {
-    val keyEntity = pgpKeyDetails.toKeyEntity(accountEntity).copy(
-      source = KeyImportDetails.SourceType.NEW.toPrivateKeySourceTypeString(),
-      privateKey = KeyStoreCryptoManager.encryptSuspend(pgpKeyDetails.privateKey).toByteArray(),
-      storedPassphrase = KeyStoreCryptoManager.encryptSuspend(passphrase)
-    )
-
-    if (roomDatabase.keysDao().insertSuspend(keyEntity) == -1L) {
-      throw NullPointerException("Cannot save a generated private key")
-    }
-  }
```

Comment:
Was this unused?

Reply:
Yes. It's unused. I kept this code for some time but after reviewing this class I've decided to remove it.